i18n/de.json

```diff
@@ -31,6 +32,8 @@
   "next": "Nächstes",
   "search...": "Suche...",
 
+  "edit user": "Editiere Benutzer",
+
   "registration prompt": "Geben Sie ihre Daten ein, um sich zu registrieren",
   "username taken": "Benutzername ist bereits belegt",
   "name prompt": "Geben Sie Ihren Namen ein",
```

Comment:
Translations for the buttons "abort" and "save" are missing. "Bearbeiten" might be a better translation for "edit" than "editieren"